### PR TITLE
[Images] How to create custom API token for Images

### PR DESCRIPTION
**Pull request by marciocloudflare**
- Added details for creating a custom token for images

### CODE REVIEW

File: content/images/cloudflare-images/tutorials/integrate-cloudflare-images.md [hunk 1]
```diff
@@ -332,18 +332,4 @@ This refers to your Cloudflare Image account ID. You can get this value in your
...
-5. Give your custom token a name.
-6. In **Permissions**, click the _Select item..._ drop-down menu next to _Account_. Choose _Cloudflare Images_. In the next drp-down menu, choose _Edit_.
-
-![How to create a custom token for Cloudflare images](/images/static/tutorials/integrate-cloudflare-images/step-02-custom-token-setup.jpg)
-
-7. Click **Continue to summary** > **Create Token**.
-
-Your token is now created. Copy it and keep it in a safe place. It grants access to your Cloudflare Images account.
-
+This value refers to a Cloudflare API token, with `read` and `update` permissions for Cloudflare Images. If you do not have one yet, refer to [Make your first API request](/images/cloudflare-images/api-request/) to learn how to create a Cloudflare API token.
```
Comment (angelampcosta) [inline content/images/cloudflare-images/tutorials/integrate-cloudflare-images.md]: In the other file, read and update permissions are capitalized. I usually capitalize, but I'm not sure which one is correct.

File: content/images/cloudflare-images/api-request.md [hunk 2]
```diff
@@ -23,7 +23,23 @@ account and website.
...
 
-Refer to [Creating API tokens](/api/tokens/create/#getting-started) to learn how to create a custom token.
+To use Cloudflare Images you need to create a custom token with the correct `Read` and `Update` permissions:
+
+1. In the Cloudflare dashboard, locate [API Tokens](https://dash.cloudflare.com/profile/api-tokens) under **My Profile** > **API Tokens**.
+2. Click **Create Token**.
+3. In Custom token, click **Get started**.
+4. Give your custom token a name.
+5. Scroll to **Permissions**.
+6. Click the _Select item..._ drop-down menu next to _Account_ and choose _Cloudflare Images_.
```
Comment (angelampcosta) [inline content/images/cloudflare-images/api-request.md]: When you do this step Account and Cloudflare images are already there? You don't need to select them in this step, right?

**Review by angelampcosta — APPROVED** [review]

File: content/images/cloudflare-images/tutorials/integrate-cloudflare-images.md (hunk 1 above)
Comment (marciocloudflare) [reply content/images/cloudflare-images/tutorials/integrate-cloudflare-images.md]: updated

File: content/images/cloudflare-images/api-request.md (hunk 2 above)
Comment (marciocloudflare) [reply content/images/cloudflare-images/api-request.md]: Good catch! Thanks :)

**Pull request merged**